yjit/src/codegen.rs

```diff
@@ -9769,6 +9839,9 @@ pub struct CodegenGlobals {
 
     /// Page indexes for outlined code that are not associated to any ISEQ.
     ocb_pages: Vec<usize>,
+
+    /// Lazily push a frame when it calls rb_yjit_check_pc() with a PC in this HashMap
```

Comment:
```suggestion
    /// Map of cfunc YARC PCs to CMEs, used to
    /// lazily push a frame when rb_yjit_check_pc() is called with a PC in this HashMap
```

Reply:
applied at https://github.com/ruby/ruby/pull/10080/commits/f80b64a74043282b4f2dc3ee0923ad8563c215a8 (and https://github.com/ruby/ruby/pull/10080/commits/9372f4f93cd8128d320c5cb7f558fd3492e5e5d8)